src/gradata/hooks/_base.py

```
@@ -0,0 +1,97 @@
+
+
+def extract_message(data: dict) -> str | None:
+    """Extract user message from hook stdin data."""
+    msg = data.get("message") or data.get("prompt") or data.get("content") or ""
+    if not isinstance(msg, str):
+        return None
+    msg = msg.strip()
+    return msg if msg else None
+
+
+def get_brain() -> "Brain | None":
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)